### DISCUSSION TITLE
Lodash 的使用

### DISCUSSION THREAD

**Discussion by FoundTheWOUT** [bangumi/frontend#128] [Q&A]
请问是有考虑使用 lodash 这个库吗，我看现在有安装这个库的类型，但没安装这个库。
https://github.com/bangumi/frontend/blob/c850dd9161d5a48a25ca873b78f9c100c5055569/packages/website/package.json#L28

**Comment by cokemine** [comment]
怪

**Comment by Ayase-252** [comment]
怪了为啥，谁写的（我啊，那没事了

去掉看看能不能跑呢？

**Reply by FoundTheWOUT** [reply]
要不考虑用一下这个库？挺实用的

**Reply by Ayase-252** [reply]
也可以

**Reply by FoundTheWOUT** [reply]
我在想会不会和 [这个](https://www.typescriptlang.org/tsconfig#skipLibCheck) 有关
> In these cases, you should consider using a feature like [yarn’s resolutions](https://yarnpkg.com/lang/en/docs/selective-version-resolutions/) to ensure there is only one copy of that dependency in your tree or investigate how to ensure there is only one copy by understanding the dependency resolution to fix the issue without additional tooling.

我在做 #127 的时候遇到了很多库依赖的错误，就是因为不同的库依赖了很多相同的，但版本却不一样的库。后来把 skipLibCheck 开了就好了。这部分具体是怎么解决

**Comment by trim21** [comment]
已经在用lodash了，这应该不再是个问题了...
